tests/sentry/workflow_engine/endpoints/test_organization_workflow_index.py

```diff
@@ -26,3 +27,104 @@ def test_simple(self):
     def test_empty_result(self):
         response = self.get_success_response(self.organization.slug)
         assert response.data == []
+
+
+class OrganizationWorkflowCreateAPITestCase(OrganizationWorkflowAPITestCase):
+    method = "POST"
+
+
+@region_silo_test
+class OrganizationWorkflowCreateTest(OrganizationWorkflowCreateAPITestCase):
```

Comment:
should this test that an invalid workflow returns the correct error response?

also do we expect to use `OrganizationWorkflowCreateAPITestCase` elsewhere?

Reply:
yeah, i should :'( i'll fix up real quick before mergin' anything.